web/react/components/mention_list.jsx

```diff
@@ -127,17 +128,20 @@ module.exports = React.createClass({
         }
         var numMentions = Object.keys(mentions).length;
 
-        if (numMentions < 1) return (<div/>);
+        if (numMentions < 1) return null;
 
-        var height = (numMentions*36) + 4;
-        var width = $('#'+this.props.id).parent().width();
-        var bottom = $(window).height() - $('#'+this.props.id).offset().top;
-        var left = $('#'+this.props.id).offset().left;
-        var max_height = $('#'+this.props.id).offset().top - 10;
+        var $mention_tab = $('#'+this.props.id);
+        var maxHeight = Math.min(303, $mention_tab.offset().top - 10);
```

Comment:
Also can you change that 303 to 292px, would've done that on my own but since you've already opened a pull request addressing the mentions stuff, can you include that small change in this pull request.